package.json

```diff
@@ -24,9 +24,10 @@
   "engines": {
     "node": "^8.1 || >=10.*"
   },
-  "main": "lib/stripe.js",
+  "main": "src/stripe.js",
```

Comment:
I don't think we want to change that. That's the entry point of the shipped library which should still be in "lib" directory.

Reply:
Oops thanks for catching! Just updated